build.gradle.kts

```diff
@@ -215,7 +215,7 @@ val userPassword = "Passw0rd@1"
 
 //Identity Cloud Admin Credentials
 val adminUsername = "amadmin"
-val adminPassword = "LujHa2N1lzcgzYNrLQMbVRvr"
```

Comment:
This will dissapear when the job to create user data as part of the pipeline happens?

Reply:
Yes, I will remove it when I will create the user directly in the pipeline.

Reply:
I have added dummy values for username and password.

Reply:
You miss update the value of `userPassword` 😉

Reply:
That it is a dummy value, the user can be created with any other password. I can also add "password" if you think that's better.

Reply:
No, no, that's ok!